library/network/src/modules/CWMFirewallInterfaces.rb

```diff
@@ -1095,7 +1095,7 @@ def CreateOpenFirewallWidget(settings)
       end
       firewall_settings = VBox(
         Frame(
-          _("Firewall Settings"),
+          _("Firewall Settings for #{SuSEFirewall.firewall_service}"),
```

Comment:
You mean in terms of translation? Because IIRC I can see the updated string in the CWMFirewallInterfaces consumers.